app/controllers/roadmap/course-extension-ideas.ts

```diff
@@ -23,6 +25,14 @@ export default class CourseExtensionIdeasController extends Controller {
   @service declare authenticator: AuthenticatorService;
   @service declare store: Store;
 
+  get courseExtensionIdeas() {
```

Comment:
Just use `@model.courseExtensionsIdeas` directly in the template, the indirection here isn't worth it